src/api/app/models/token/workflow.rb

```diff
@@ -11,6 +11,8 @@ def call(options)
 
     scm_extractor_payload = extractor.call
     yaml_file = Workflows::YAMLDownloader.new(scm_extractor_payload).call
+    raise 'workflows.yml is missing' if yaml_file.blank?
```

Comment:
When this happens, do we get the error in the webhook response on the SCM? If not, this won't help out users to know what is wrong.

Reply:
yes https://github.com/openSUSE/open-build-service/pull/11321/files#diff-184bebd6d68042ab1a0659db7e98ad054cf2384141156b5c0d2f6ef9eed6931aR43